### PR TITLE
[Master][DEL] incorrect commits

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Description of the issue/feature this PR addresses:
This PR deletes changes done in previous versions that should not be ported to `master` branch.

Current behavior before PR:
`master` branch contains changes that should not have.

Desired behavior after PR is merged:
`master` branch have all the desired changes.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

cc: @KangOl @mart-e

### CODE REVIEW

File: addons/account/static/src/js/reconciliation/reconciliation_model.js [hunk 1]
```diff
@@ -1044,8 +1044,7 @@ var StatementModel = BasicModel.extend({
      * @returns {object}
      */
     _formatToProcessReconciliation: function (line, prop) {
-        // Do not forward port in master. @CSN will change this
```
Comment (KangOl) [inline addons/account/static/src/js/reconciliation/reconciliation_model.js]: @csnauwaert is it safe to revert it now in master?
I would leave it until you fix it correctly.

File: odoo/addons/base/models/ir_module.py [hunk 2]
```diff
@@ -305,14 +305,6 @@ def unlink(self):
...
                 raise UserError(_('You try to remove a module that is installed or will be installed'))
         self.clear_caches()
-        # Installing a module creates entries in base.module.uninstall, during
-        # the unlink process of ir.module.module we try to update the
-        # base.module.uninstall table's module_id to null, which violates a
-        # non-null constraint, effectively raising an Exception.
-        # V11-only !!DO NOT FORWARD-PORT!!
-        self.env['base.module.uninstall'].search(
-            [('module_id', 'in', self.ids)]
-        ).unlink()
```
Comment (Elkasitu) [inline odoo/addons/base/models/ir_module.py]: see 07510cd

File: addons/account/static/src/js/reconciliation/reconciliation_model.js (hunk 1 above)
Comment (csnauwaert) [reply addons/account/static/src/js/reconciliation/reconciliation_model.js]: @KangOl The branch with the fix is not yet finished and this is not fixed yet in master, you can forward port it, I'll revert/fix it within my branch

**Comment by mreficent** [comment]
@KangOl I updated the commit to just delete the redundant part of `ir_module`

**Pull request merged**